engine/runtime/src/main/scala/org/enso/compiler/data/BindingsMap.scala

```diff
@@ -937,6 +940,25 @@ object BindingsMap {
 
     override def qualifiedName: QualifiedName =
       module.getName.createChild(symbol.name)
+
+    def findExportedSymbolFor(
+      name: String
+    ): org.enso.compiler.data.BindingsMap.ResolvedName =
+      ResolvedPolyglotField(this, name)
+  }
+
+  case class ResolvedPolyglotField(symbol: ResolvedPolyglotSymbol, name: String)
```

Comment:
minor: None of the FQN in method signatures is necessary here. Moreover, add a space between method definitions. Could have been handled by `scalafmt` though.